charts/consul/test/terraform/gke/main.tf

```
@@ -24,14 +24,33 @@ resource "google_container_cluster" "cluster" {
   location           = var.zone
   min_master_version = data.google_container_engine_versions.main.latest_master_version
   node_version       = data.google_container_engine_versions.main.latest_master_version
-
+  node_config {
+    tags = ["consul-k8s-${random_id.suffix[count.index].dec}"]
+  }
   pod_security_policy_config {
     enabled = true
   }
 
   resource_labels = var.labels
 }
 
+resource "google_compute_firewall" "firewall-rules" {
+  project     = var.project
+  name        = "consul-k8s-acceptance-firewall-${random_id.suffix[count.index].dec}"
+  network     = "default"
+  description = "Creates firewall rule targeting tagged instances"
```

need to update description I think to reflect what it does
```suggestion
  description = "Creates firewall rule allowing traffic from nodes and pods of the ${random_id.suffix[count.index == 0 ? 1 : 0].dec} Kubernetes cluster."
```